std_msgs/String map_name
---
grid_planner/GridMap grid_map